docs/learn/learn-bridge-hub.md

```diff
@@ -25,6 +29,11 @@ A two-way trustless bridge between chains A and B can be viewed as two one-way b
 source and a target chain. Any bridge operating on the Bridge Hub will have on-chain (pallets) and
 offchain (relayers) components.
 
+Examples of trustless on Bridge Hub include:
+
+- Snowfork, a decentralized bridge between Polkadot and Ethereum
```

Comment:
should we specify that this is an Ethereum light client on Polkadot that syncs to Ethereum and keeps the Ethereum state up-to-date on Polkadot?

Reply:
Maybe this is good for the next PR, we can add those details in the followup 👍